extra/sponge.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
```

Comment:
Changed. It should be mentioned in the [Contributing Guidelines](https://docs.spongepowered.org/en/latest/devs/guidelines/) as a difference from Google's style guide